packages/material-ui-lab/src/Slider/Slider.test.js

```diff
@@ -184,4 +184,28 @@ describe('<Slider />', () => {
       assert.strictEqual(trackAfter.prop('style').width, 'calc(100% - 5px)');
     });
   });
+
+  describe('prop: step', () => {
+    const transitionComplexDuration = 375;
+    let wrapper;
+    let clock;
+
+    before(() => {
+      clock = useFakeTimers();
+      wrapper = mount(<Slider value={0} min={0} max={102} step={10} />);
+    });
+
+    after(() => {
+      clock.restore();
+    });
+
+    it('after change value should change position of thumb', () => {
+      wrapper.setProps({ value: 103 });
+
+      clock.tick(transitionComplexDuration);
+
```

Comment:
This test depends on the theme now. Setting the props immediately changes the style. The css just provides a transition. I don't think enzyme can actually emulate that? The clock should therefore be irrelevant.

It also passes without the change in the implementation.